app/scripts/modules/kubernetes/src/pipeline/stages/runJob/configureJob.html

```diff
@@ -0,0 +1,115 @@
+                                field="account"
+                                provider="'kubernetes'"
+                                accounts="ctrl.accounts"
+                                on-change="ctrl.updateNamespaces()">
+          </account-select-field>
+        </stage-config-field>
+      
+        <stage-config-field label="Namespace" help-key="kubernetes.namespace">
+          <namespace-select-field component="ctrl.stage"
+                                  field="namespace"
+                                  hide-label="true"
+                                  columns="12"
+                                  account="ctrl.stage.account"
+                                  namespaces="ctrl.namespaces">
+          </namespace-select-field>
+        </stage-config-field>
+    
+        <stage-config-field label="Image">
+          <ui-select name="containerSelect" ng-model="ctrl.stage.container" class="form-control input-sm" required>
+              <ui-select-match>{{ ctrl.stage.container.imageDescription | kubernetesImageId }}</ui-select-match>
+              <ui-select-choices
+                repeat="container in ctrl.containers | orderBy: 'imageDescription.imageId'"
+                group-by="ctrl.groupByRegistry"
+                refresh-delay="0"
+                ui-disable-choice="container.message"
+                refresh="ctrl.searchImages($select.search)">
+                <span ng-bind-html="container.imageDescription | kubernetesImageId | highlight: $select.search"></span>
+              </ui-select-choices>
+            </ui-select>
+        </stage-config-field>
+
+        <stage-config-field label="Container Name">
+            <div>
+              <input type="text" class="form-control input-sm" ng-model="ctrl.stage.container.name">
+            </div>
+          </stage-config-field>
+      </v2-wizard-page>
+      <v2-wizard-page label="Container" done="true">
+        
+        <div class="sm-label-left">
+          <b>Commands</b>
+        </div>
+        <kubernetes-container-commands commands="ctrl.stage.container.command">
+        </kubernetes-container-commands>
+        
+      
+        <div class="sm-label-left">
+          <b>Args</b>
+        </div>
+        <kubernetes-container-arguments args="ctrl.stage.container.args">
+        </kubernetes-container-arguments>
+      
+        <div class="sm-label-left">
+          <b>Environment Variables</b>
+        </div>
+        <kubernetes-container-environment-variables env-vars="ctrl.stage.container.envVars">
+        </kubernetes-container-environment-variables>
+
+        <div>
+          <div class="sm-label-left">
+            <b>Volumes</b>
+          </div>
+          <kubernetes-container-volumes volume-sources="ctrl.stage.volumeSources" volume-mounts="ctrl.stage.container.volumeMounts">
+          </kubernetes-container-volumes>
+        </div>
+      </v2-wizard-page>
+
+      <v2-wizard-page label="Advanced Settings" done="true">
```

Comment:
Can't see from the screenshots, but I'm guessing this section has the same problem.